python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -564,7 +564,11 @@ def _addReferenceLines(self, ax, panel, panel_range, meds, legend_font_size=7):
                 reference_value = self.panels[panel].referenceValue
                 reference_label = "${{\\mu_{{ref}}}}$: {:10.3F}".format(reference_value)
             ax2.axvline(reference_value, ls="-", lw=1, c="black", zorder=0, label=reference_label)
-        if self.panels[panel].histDensity:
+        if (
+            self.panels[panel].histDensity
+            and (panel_range[1] - panel_range[0] != 0)
```

Comment:
Is there a reason to write this instead of `panel_range[0] != panel_range[1]`? I suppose they'll end up doing the same thing even with nans.